contrib/oauth2-proxy_autocomplete.sh

```diff
@@ -20,6 +20,10 @@ _oauth2_proxy() {
 			COMPREPLY=( $(compgen -W "google azure facebook github keycloak gitlab linkedin login.gov digitalocean" -- ${cur}) )
 			return 0
 			;;
+		-real-client-ip-header)
```

Comment:
I'm unfamiliar with`@` as a wildcard in switch statements:
```suggestion
		-@real-client-ip-header)
```

or should I just:
```suggestion
		--real-client-ip-header)
```

Reply:
I think the @ is like an optional, but we aren't allowing single - anymore, so I think double dash is the right way to go here